WoWsShipBuilder.Web/Components/DepthChargeDamageDistributionChart.razor

```diff
@@ -0,0 +1,26 @@
+﻿@using WoWsShipBuilder.Web.Services
+@using WoWsShipBuilder.Core.DataContainers
+@using WoWsShipBuilder.DataStructures
+@inject DepthChargeDamageDistributionChartInterop DepthChargeInterop
+
+
+<MudPaper Outlined="true">
+    <canvas id="@("DepthCharges_" + ShipIndex)" height="350" width="350"></canvas>
+</MudPaper>
+
+@code {
+
+    [Parameter]
+    public string ShipIndex { get; set; } = default!;
+    [Parameter]
+    public DepthChargeDamageDistributionChartRecord? DataRecord { get; set; }
+    
+    protected override async Task OnAfterRenderAsync(bool firstRender)
+    {
```

Comment:
Does this have to be redrawn after every single render loop? 
Ensure this is not called more than necessary, ideally, only once.